telegram/ext/jobqueue.py

```diff
@@ -195,6 +210,46 @@ def run_repeating(self, callback, interval, first=None, context=None, name=None)
         self._put(job, time_spec=first)
         return job
 
+    def run_monthly(self, callback, when, day, context=None, name=None):
+        """Creates a new ``Job`` that runs on a monthly basis and adds it to the queue.
+
+        Args:
+            callback (:obj:`callable`): The callback function that should be executed by the new
+                job. It should take ``bot, job`` as parameters, where ``job`` is the
+                :class:`telegram.ext.Job` instance. It can be used to access its ``Job.context``
```

Comment:
This callback signature is deprecated. Please update to v12 compatible docstring (have a look at e.g. the [`Handler docs`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.handler.html)).
While you're at it, could you do so for the other `run_*` methods as well?

Reply:
Never mind, fixed in #1642

Reply:
okay, done!